### DISCUSSION TITLE
Participants section lock property

### DISCUSSION THREAD

**Discussion by oksuzgil** [mynaparrot/plugNmeet-server#287] [General]
The following permissions are not working in the Participants section.

lock the whiteboard
lock chat

![image](https://github.com/mynaparrot/plugNmeet-server/assets/73983839/ace1dabf-3888-4517-a59d-6595d8c20b64)

**Comment by jibon57** [comment]
Make sure other user isn't admin

**Comment by oksuzgil** [comment]
I checked, it's not admin.

Also the shared notebook in the same menu does not locked. (etherpad)

**Reply by jibon57** [reply]
working perfectly from my end

**Comment by oksuzgil** [comment]
Plugnmeet Demo Page Record

https://github.com/mynaparrot/plugNmeet-server/assets/73983839/dd0ef704-88ff-4f4c-b01b-adb4dce45741

**Reply by jibon57** [reply]
You are misunderstanding the meaning of lock for whiteboard & shared notepad. This is way we've designed to make it **read only** mood if locked, not disabled.

**Reply by oksuzgil** [reply]
Yes it makes sense. It becomes read-only. But when I type "lock" and other lock functions lock the corresponding button, it is thought to be locked in this way.

I think the lock operation should lock the relevant button, the read-only operation should do the read-only operation as it is now.

In addition, when creating a room for those who will only provide video calls, these buttons in the bottom menu should be completely removed.

Thank you.
